src/atoms/affine/transpose.jl

```diff
@@ -139,3 +139,6 @@ end
 
 ctranspose(x::AbstractExpr) = CTransposeAtom(x)
 ctranspose(x::Constant) = Constant(x.value')
```

Comment:
`ctranspose` isn't a thing anymore, so this can be removed.

Reply:
We'll just need to add AdjointAtom instead or something